build/helper/helper.py

```diff
@@ -42,7 +42,7 @@ def camelcase_to_snakecase(camelcase_string):
     '''Converts a camelCase string to lower_case_snake_case'''
     # https://stackoverflow.com/questions/1175208/elegant-python-function-to-convert-camelcase-to-snake-case
     s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', camelcase_string)
-    return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
+    return re.sub('([a-z])([A-Z])', r'\1_\2', s1).lower()
```

Comment:
I think I'm okay with this, given the lack of changes to our other APIs, but note that you could also have set `python_name` for any affected functions.

I wouldn't mind seeing a unit test added to test_helper.py.